spark-jobs/src/main/scala/za/co/absa/enceladus/common/SparkCompatibility.scala

```diff
@@ -27,5 +27,6 @@ import za.co.absa.commons.version.impl.SemVer20Impl.SemanticVersion
 
 object SparkCompatibility {
   val minSparkVersionIncluded: SemanticVersion = semver"2.4.2"
+  val minSparkHyperConformanceIncluded: SemanticVersion = semver"2.4.3"
```

Comment:
Sorry guys, a post-merge comment.
The requirement for HyperConformance is the same as normal Conformance.
I'd say, 2.4.2 for both. Adrian was asking me in the chat, I haven't guessed the context.
We can fix this some other time.
Sorry.

Reply:
Should be fine, at least we have precedence for the next time.